api/v1alpha1/project.go

```diff
@@ -23,9 +23,14 @@ type ProjectMetadata struct {
 
 type ProjectSpec struct {
 	// Owners is the list of identifiers of owners of the project.
-	// It can be any strings that uniquely identifies the owners, such as email addresses or Jac group identifiers.
+	// It can be any string that uniquely identifies the owners, such as email addresses or Jac group identifiers.
 	Owners []string `yaml:"owners,omitempty" json:"owners,omitempty"`
 
+	// CodeOwners is the list of GitHub Code Owners of the project.
+	// This gets added to the CODEOWNERS file from the repository.
+	// Unlike Owners above which are based on Jac, they are GitHub usernames or teams.
+	CodeOwners []string `yaml:"codeOwners,omitempty" json:"codeOwners,omitempty"`
```

Comment:
What's the reason for allowing code owners to be specified in `Project` resource, in addition to `CODEOWNERS` file? We should have a single source of truth for code owners, no? It's OK to have internal fields like this that we populate in memory, but in such case I don't make them serializable.

Reply:
My thought was to be able to set the teams or streams explicitly on top of the `CODEOWNERS`. I initially wanted to take the `owners` field, but it doesn't translate perfectly into GitHub teams or people. It can go away if anybody feels strongly about it.